tox-wheel is not needed to run tests.
tox version pin is too strict.

--- incremental-24.7.2/tox.ini.orig
+++ incremental-24.7.2/tox.ini
@@ -2,8 +2,6 @@
 minversion = 3.23.0
 requires =
     virtualenv >= 20.4.3
-    tox < 4.0.0
-    tox-wheel >= 0.6.0
 isolated_build = true
 envlist =
     flake8,
